apps/remote_control/lib/lexical/remote_control/completion.ex

```diff
@@ -18,4 +20,42 @@ defmodule Lexical.RemoteControl.Completion do
       |> Enum.map(&Candidate.from_elixir_sense/1)
     end
   end
+
+  def struct_fields(%Document{}, %Position{character: nil}) do
+    []
+  end
+
+  def struct_fields(%Document{} = document, %Position{} = position) do
+    context =
+      Code.Fragment.surround_context(
+        Document.to_string(document),
+        {position.line, position.character}
+      )
+
+    case matched_alias(context) do
+      {:ok, struct_alias} ->
+        struct_module = Ast.expand(document, position, struct_alias)
+
+        fields =
+          if function_exported?(struct_module, :__struct__, 0) do
+            for {field_name, _v} <- Map.from_struct(struct_module.__struct__()), do: field_name
+          end
+
+        List.wrap(fields)
```

Comment:
if it's not a struct, this will return `[nil]` that doesn't seem to be desired, right?